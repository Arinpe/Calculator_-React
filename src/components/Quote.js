export default function Quote() {
  return (
    <p className="quote">
      “Common sense is calculation applied to life. ”
      ― Henri Frederic Amiel
    </p>
  );
}
