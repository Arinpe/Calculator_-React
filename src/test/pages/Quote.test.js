import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Quote from '../../components/App';

describe('Quote', () => {
  const { asFragment } = render(<Quote />);

  test('should render without crashing', () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
