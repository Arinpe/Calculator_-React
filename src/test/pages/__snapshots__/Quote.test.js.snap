// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote should render without crashing 1`] = `
<DocumentFragment>
  <div
    class="nav"
  >
    <h1
      class="logo"
    >
      Math Magicians
    </h1>
    <div
      class="nav-links"
    >
      <a
        class="nav-link"
        href="/"
      >
        Home
      </a>
      <a
        class="nav-link"
        href="/calculator"
      >
        Calculator
      </a>
      <a
        class="nav-link"
        href="/quote"
      >
        Quote
      </a>
    </div>
  </div>
  <div
    class="home"
  >
    <h1>
      Welcome to our page!
    </h1>
    <p>
      <span>
        A calculator is a device that performs arithmetic operations.
      </span>
    </p>
  </div>
</DocumentFragment>
`;
